zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2523,6 +2523,57 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /users/{userKey}/authorizations/search:
+    post:
+      tags:
+        - Authorization
+      summary: "Query user authorizations (alpha)"
+      description: |
+        Search for user authorizations based on given criteria.
+
+        :::note
+        This endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
+        See the [Camunda 8 REST API overview](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api)
+        for further details.
+        :::
```

🚀 Thanks for adjusting the spec directly 🦖 

🔧 Can we adjust this description as well for consistency?
```suggestion
        Note that this endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
        The Camunda 8 API (REST) Overview page provides further details.

        :::note
        This endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and may be subject to change
        in future releases.
        :::
```